### DISCUSSION TITLE
目前也没多少新增，讨论也没啥热度了，看来已经停工的确实没预想中的那么多

### DISCUSSION THREAD

**Discussion by pacoblack** [WeNeedHome/SummaryOfLoanSuspension#1116] [General]
看了下十几天就新增了一个

**Comment by kongji123** [comment]
应该很多。很多不知道这里统计

**Comment by Zoey-nn** [comment]
维权太难了，业主自己建群讨论还要提防政府插几个眼线进来，在里面带节奏把方向跑偏，顺便再让你们起内讧。反正他们的目的就是让你们无法发声，或者拖住你们不要把事情闹大。这帮人真的太贱了…

**Reply by kajsnlk** [reply]
没有隐私的后果，腾讯也是助纣为虐

**Comment by clleo888** [comment]
目前也没多少新增，讨论也没啥热度了:
可能一: 看来已经停工的确实没预想中的那么多
可能二: 看来政府对负面消息的打压已经去到你想像不到的地步了

我比较相信可能二。

**Comment by itsoursduty** [comment]
身边就有一个 业主联合起来谈判的。 有烂尾风险的
